src/components/Overlay/Overlay.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import {block} from '../utils/cn';
+
+import './Overlay.scss';
+
+const b = block('overlay');
+
+export type OverlayBackground = 'base' | 'float';
+
+export interface OverlayProps {
+    className?: string;
+    view?: OverlayBackground;
+    visible?: boolean;
+    children?: React.ReactNode;
+}
+
+export function Overlay({
```

Comment:
@amje @korvin89  Do we want to name this component `Overlay` or `Backdrop`?

Reply:
Backdrop is a special case of Overlay use. So I think Overlay name is ok